libs/coin-modules/coin-polkadot/src/network/sidecar.ts

```diff
@@ -136,14 +137,7 @@ const fetchStakingInfo = async (addr: string): Promise<SidecarStakingInfo> => {
  * @returns {SidecarNominations}
  */
 const fetchNominations = async (addr: string): Promise<SidecarNominations> => {
```

Comment:
Do we really want to keep this method ? Maybe we need to edit directly where it's used with node.fetchNominations call as now it doesn't relate to sidecar anymore

Reply:
I've removed the method. Thanks